### DISCUSSION TITLE
Disable 'always hidden'

### DISCUSSION THREAD

**Discussion by purekimwater** [jordanbaird/Ice#183] [General]
Hi, I'd like to disable the 'always hidden' section of the menu bar.

New icons always get added to the most left side of it,
which are automatically always hidden.

Is there a way to disable it? or maybe could it be added in the future?
Thanks.

**Comment by scottrbaxter** [comment]
Was going to suggest exactly this. Apparently i change icons regularly enough that this is becoming a problem. I really don't need an "always hidden" section at all... hover to reveal hidden icons is good enough for me. Given newly added icons always go into the "always hidden" area means they're forever being hidden unexpectedly. Simply toggling Ice's `Show Ice icon` on puts this app's icon into "always hidden", which will easily confuse users.

It's a cool feature for people that want to keep a service running but never ever want to see the associated icon, ever... but given the system limitation of new icons being added here makes it seem like this shouldn't even be enabled by default, though keeping it around for users that absolutely want it would be a good compromise.

**Comment by jordanbaird** [comment]
Just implemented this in 2ebd87a994be406ab4f45667b3b969f104ad6c6a
